build/standalone-install.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-# VERSION 0.3 by d3vilh@github.com aka Mr. Philipp.
+# VERSION 0.4 by damian@bytefarm.ch
 #
 
 # All the variables
-GOVERSION="1.22.3"
+GOVERSION="1.21.5"
```

Comment:
This should match the go version from the latest release version, instead of downgrading.